src/universal/components/CreateAccountButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import styled from 'react-emotion'
+import SecondaryButton from 'universal/components/SecondaryButton'
+import {RouteComponentProps, withRouter} from 'react-router-dom'
+
+const StyledButton = styled(SecondaryButton)({
+  paddingLeft: 16,
+  paddingRight: 16
+})
+
+const CreateAccountButton = (props: RouteComponentProps) => {
+  const {history} = props
+  const handleClick = () => history.push('/create-account?from=demo')
+  return <StyledButton onClick={handleClick}>{'Create Free Account'}</StyledButton>
```

Comment:
what if the viewer is logged in? Either this should say "Go to dashboard" or it shouldn't exist at all

Reply:
Agreed, I’ll look for the a to specify that logic.

Reply:
@mattkrick not immediately sure on how to do that, so if you see this before I figure it out, I’ll probably need a hint

Reply:
i'd just check local storage to see if there's a token present. if so, they're logged in